### PR TITLE
Fixed the url for Broken logo 

### PR DESCRIPTION
**Pull request by cg-cnu**
## Description

Url returning 503, fixed it by changing it to rawgit.

## Motivation and Context

Fixes https://github.com/siddharthkp/bundlesize/issues/125

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by cg-cnu** [comment]
@kuldeepkeshwar Thanks for the review! I already changed those files as well. Total of 3 files changed. 
Can you check and let me know! 🙂

**Comment by kuldeepkeshwar** [comment]
This is still missing
https://github.com/siddharthkp/bundlesize/blob/a85f938c116b21d1d171500d1a1c37c7eea8956e/README.md#L86

**Comment by ForsakenHarmony** [comment]
also not sure if we need it in the readme, but ig it won't do any harm

**Comment by palashmon** [comment]
Sidenote: Please don't add Sid as reviewer until further notice, as he is on break for sometime and had unwatched this repo. So, most probably he will not be able to review anything, as mentioned in #121 

Regards!

**Comment by cg-cnu** [comment]
@kuldeepkeshwar Sorry for tagging sid! I completely forgot. I modified the `build status` in `readme.md` to `rawgit`.
@ForsakenHarmony Removed the package-lock from the commit. 
Thanks! 😃

**Comment by kuldeepkeshwar** [comment]
@cg-cnu can you check it again as I can still see `package-lock.json` in diff

**Comment by cg-cnu** [comment]
@kuldeepkeshwar I thought deleting the file and commiting again will help. I guess I created some conflicts. Can you guide me on this one. Sorry for the trouble. 😞

**Comment by SaraVieira** [comment]
@cg-cnu @kuldeepkeshwar I can still see the lock on the diffs

**Comment by cg-cnu** [comment]
@SaraVieira Can you point out how I can remove it ? Or you think I should scrap this and create a fresh pr ?

**Comment by SaraVieira** [comment]
@cg-cnu Are you using npm 5? 
Get the lock from master, delete all the node modules and then do npm. Install and you should be fine ☺️

**Comment by cg-cnu** [comment]
@SaraVieira Yes am on npm5. Sure will do that. Thanks 😄

**Comment by SaraVieira** [comment]
@cg-cnu Awesome! I will update danger to also check for changes in the lock and give you these instructions :)

**Comment by SaraVieira** [comment]
@cg-cnu Awesome! I'm going to approve this and as soon as it's merged I'll do a patch release 🎉

**Comment by SaraVieira** [comment]
@forsakenHarmony @kuldeepkeshwar I think it looks great now

**Comment by cg-cnu** [comment]
@SaraVieira Thanks so much for helping me out. Just trying to understand, I shouldn't push the package-lock.json (even if it has changes) unless I made any changes in the package.json and npm installs.

**Comment by SaraVieira** [comment]
@cg-cnu I believe so, I will enable greenkeeper in this repo so that we can get warnings when there new releases of stuff so that we are always up to date and that does not happen again :)

**Comment by kakadiadarpan** [comment]
@SaraVieira Shouldn't we add `package-lock.json` to `.gitignore`?

And when we update any package or add a new package, we can explicitly add `package-lock.json` in the commit.

**Comment by SaraVieira** [comment]
@kakadiadarpan we could but that might bring issues in the future in Travis :/

**Comment by reznord** [comment]
we don't even need `package-lock.json` file. And installing deps on travis hardly takes a minute. So, there is no point in adding either `package-lock.json` or `yarn.lock` files to the repo. 

I will be opening an issue for the same to discuss on it.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img src="https://raw.githubusercontent.com/siddharthkp/bundlesize/master/art/logo.png" height="200px"/>
+  <img src="https://cdn.rawgit.com/siddharthkp/bundlesize/master/art/logo.png" height="200px"/>
```
Comment (reznord) [inline README.md]: `cdn.rawgit` redirects to `raw.githubusercontent` itself. Please revert this change. it is not necessary.

File: README.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ This makes it great for using with applications that are bundled with another to
 
 #### 2) build status
 
-![build status](https://raw.githubusercontent.com/siddharthkp/bundlesize/master/art/status.png)
+![build status](https://cdn.rawgit.com/siddharthkp/bundlesize/master/art/status.png)
```
Comment (reznord) [inline README.md]: same here.

cdn.rawgit redirects to raw.githubusercontent itself. Please revert this change. it is not necessary.

File: store/views/auth.pug [hunk 3]
```diff
@@ -5,7 +5,7 @@
   <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
   </head>
   <body>
-    <img class="logo" src="https://raw.githubusercontent.com/siddharthkp/bundlesize/master/art/logo.png"/>
+    <img class="logo" src="https://cdn.rawgit.com/siddharthkp/bundlesize/master/art/logo.png"/>
```
Comment (reznord) [inline store/views/auth.pug]: same here!

File: store/views/build.pug [hunk 4]
```diff
@@ -5,7 +5,7 @@
   <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
   </head>
   <body>
-    <img class="logo" src="https://raw.githubusercontent.com/siddharthkp/bundlesize/master/art/logo.png"/>
+    <img class="logo" src="https://cdn.rawgit.com/siddharthkp/bundlesize/master/art/logo.png"/>
```
Comment (reznord) [inline store/views/build.pug]: same here!

**Comment by cg-cnu** [comment]
@reznord Wait! I am totally lost. I guess the request was to move the urls from `raw.githubusercontent` to `cdn.rawgit`. Am I missing something here 😕

**Comment by reznord** [comment]
> I guess the request was to move the urls from raw.githubusercontent to cdn.rawgit

right. but if you see, `cdn.rawgit` redirects to `raw.githubusercontent` itself.

**Comment by cg-cnu** [comment]
The purpose of using rawgit seems to be to get correct [`content-type`](https://github.com/rgrove/rawgit/blob/master/FAQ.md#why-is-this-necessary-cant-i-just-load-files-from-github-directly)

**Comment by SaraVieira** [comment]
@reznord @cg-cnu @kuldeepkeshwar Patch release, everyone agree ?

**Comment by kuldeepkeshwar** [comment]
@SaraVieira 👍

**Comment by cg-cnu** [comment]
@SaraVieira Sounds good 👍

**Comment by SaraVieira** [comment]
Done guys:
https://github.com/siddharthkp/bundlesize/releases/tag/v0.14.3

Thanks you so much 🎉

**Comment by paulirish** [comment]
@reznord BTW I'm asking for details about this redirect as I didn't expect this behavior. :) https://github.com/rgrove/rawgit/issues/2

**Comment by reznord** [comment]
Cool! Will keep a track of it :)
